DataFormats/Detectors/TRD/include/DataFormatsTRD/Constants.h

```diff
@@ -50,7 +50,7 @@ constexpr int NCOLMCM = 18;     // the number of pads per MCM
 
 constexpr int NBITSTRKLPOS = 11;                   // number of bits for position in tracklet64 word
 constexpr int NBITSTRKLSLOPE = 8;                  // number of bits for slope in tracklet64 word
-constexpr float PADGRANULARITYTRKLPOS = 80.f;
+constexpr float PADGRANULARITYTRKLPOS = 160.f;
```

Comment:
Shouldn't this be 40?

Reply:
mmm multitiplied it by 2 in the qc to get a nice plot.  but i think you might have a point.

Reply:
did you multiply `PADGRANULARITYTRKLPOS` or `GRANULARITYTRKLPOS` which is the inverse of the former?

Reply:
ah you are indeed right. I meant to change GRANULATIRYTRKLPOS

Reply:
changed to 40

Reply:
done